submissions/ASTkach/js-dom/js/script.js

```diff
@@ -0,0 +1,495 @@
+        image: "img/pictures/uranus.png",
+        diameter: "51 118",
+        mass: "86.8 * 10<sup>24 </sup>",
+        temp: [
+            {
+                name: "Mean Temperature",
+                value: "-195",
+            },
+        ],
+        gravity: "8.7",
+        distance: [
+            {
+                name: "Min distance to Earth",
+                value: "2 600",
+                unit: "million km",
+            },
+            {
+                name: "Max distance to Earth",
+                value: "3 150",
+                unit: "million km",
+            },
+        ],
+        lengthOfDay: "17.2",
+        moons: [
+            {
+                name: "Number of Moons",
+                value: "27",
+            },
+        ],
+        altText: "blue planet uranus with light blue stains",
+    },
+    {
+        id: "8",
+        title: "Neptun",
+        text:
+            "Neptune, third most massive planet of the solar system and the eighth and outermost planet from the Sun. Because of its great distance from Earth, it cannot be seen with the unaided " +
+            "eye. With a small telescope, it appears as a tiny, faint blue-green disk. It is designated by the symbol ♆. Neptune's orbit is almost perfectly circular; as a result, its distance from " +
+            "the Sun varies comparatively little over its nearly 164-year period of revolution. Although the dwarf planet Pluto's mean distance from the Sun is greater than Neptune's, its orbit is " +
+            "so eccentric (elongated) that for about 20 years of each revolution Pluto is actually nearer the Sun than is Neptune.",
+        image: "img/pictures/neptun.png",
+        diameter: "49 528",
+        mass: "102 * 10<sup>24 </sup>",
+        temp: [
+            {
+                name: "Mean Temperature",
+                value: "-200",
+            },
+        ],
+        gravity: "11.0",
+        distance: [
+            {
+                name: "Min distance to Earth",
+                value: "4 300",
+                unit: "million km",
+            },
+            {
+                name: "Max distance to Earth",
+                value: "4 600",
+                unit: "million km",
+            },
+        ],
+        lengthOfDay: "16.1",
+        moons: [
+            {
+                name: "Number of Moons",
+                value: "14",
+            },
+        ],
+        altText: "planet neptune in the form of a light blue ball",
+    },
+];
+
+const mainBlock = document.querySelector(".main");
+
+// function to generate html code for the current planet
+// (sorry, the sun will also be a planet)
+function getCurrentPlanet(planet) {
+    solarSystem.map((item) => {
+        if (item.id == planet) {
+            const planetTitle = item.title;
+            const planetText = item.text;
```

Comment:
Once time you use destruction. Also, you can use it here.
`const { title, text, ..... } = item;`